tutorial/debezium-vitess-init/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use a temporary layer for the build stage.
+FROM vitess/base:v8.0.0 AS base
```

Comment:
Is there away to avoid using this image? It appears to be huge 😱:

vitess/base                                                  v8.0.0                 39997be88418        3 months ago        6.1GB

Reply:
That's a good point! The `vitess/base:v8.0.0 ` docker image serves as a temporary image, I copy two binaries (`/vt/bin/vtctl` and `/vt/bin/mysqlctl`) from this temporary image to the final image. So the final image this Dockerfile creates isn't too big (~1G).

I just asked the Vitess community and see if there's any other way to do this: https://vitess.slack.com/archives/C0PQY0PTK/p1612339740175400

I'll keep you updated here.